Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/mailTeleporter.yml

```diff
@@ -0,0 +1,58 @@
+- type: entity
+  id: MailTeleporter
+  parent: BaseStructureDynamic
+  name: mail teleporter
+  description: Teleports mail addressed to the crew of this station.
+  components:
+  - type: MailTeleporter
+  - type: InteractionOutline
+  - type: Physics
+    bodyType: Static
+  - type: Transform
+    anchored: true
+    noRot: true
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+            bounds: "-0.45,-0.45,0.45,0.00"
+        density: 120
+        mask:
+        - HighImpassable
+  - type: Sprite
+    sprite: Nyanotrasen/Structures/mailbox.rsi
+    scale: 0.5, 0.5
+    layers:
+    - state: icon
+    - state: unlit
+      shader: unshaded
+      map: ["enum.PowerDeviceVisualLayers.Powered"]
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 75
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 1
+            max: 1
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: ApcPowerReceiver
+    powerLoad: 1000 # TODO if we keep this make it spike power draw when teleporting
```

Comment:
Spike the power draw

Reply:
Cargo on their way to cause brownouts just by having a mail teleporter

Reply:
:trollface:

Reply:
I'm not 100% sure how to go about spiking the draw properly though

Reply:
Are we waiting on this to merge it in? Didn't want to jump the gun or anything.

Reply:
I'm honestly not sure how to properly implement this, so we could just leave it as a todo